src/components/manage/Preferences/__snapshots__/ChangePassword.test.jsx.snap

```diff
@@ -20,178 +20,28 @@ exports[`ChangePassword renders a change password component 1`] = `
           className="ui segment"
         >
           <div
-            className="inline required field help"
+            className="password-widget"
```

Comment:
@sneridagh if this reflects a real change in the HTML/Class structure we should expect lots of breaking styles. Or is this just a follow up on the test setup changes?

Reply:
It's because of the new testing config mocks. In this case, the widgets are all mocked, to a more readable output where at a glance, you can see that the intention in the test was to just render a type of widget, and not the actual widget structure, which does not bring anything but a bunch of html. Also, when you change something from a widget, it only reflects on its test, not in a random number of tests where that widget is actually used. I guess it's more accurate and isolated than the previous state.

Reply:
So we do not expect any breaking changes whatsoever when it comes to HTML/CSS, correct?

Reply:
Nope :)